src/dmd/impcnvtab.d

```diff
@@ -49,28 +50,10 @@ TY implicitConvTy1(TY ty1, TY ty2)
     return impCnvTab.impcnvType1Tab[ty1][ty2];
 }
 
-/*************************************************
- * If ty1 and ty2 are basic types, return the TY that ty2 can
- * be implicitly converted to to bring them to a common ty.
- * Params:
- *      ty1 = first operand type
- *      ty2 = second operand type
- * Returns:
- *      ty = what ty2 should be converted to, else Terror
- */
-TY implicitConvTy2(TY ty1, TY ty2)
-{
-    return impCnvTab.impcnvType2Tab[ty1][ty2];
-}
-
 /******************************************************************************/
 
 private:
 
-immutable TY[TMAX][TMAX] impcnvResult = impCnvTab.impcnvResultTab;
-immutable TY[TMAX][TMAX] impcnvType1 = impCnvTab.impcnvType1Tab;
-immutable TY[TMAX][TMAX] impcnvType2 = impCnvTab.impcnvType2Tab;
-
 struct ImpCnvTab
 {
     TY[TMAX][TMAX] impcnvResultTab;
```

Comment:
remove the tables below this line and the code that generates them too?

Reply:
I'd like to keep this one focused on the symmetry. Once that passes, there's further simplification that can happen with the table generation.

Reply:
Noted.